pkgs/by-name/ll/llama-cpp/package.nix

```diff
@@ -0,0 +1,115 @@
+{ lib
+, cmake
+, darwin
+, fetchFromGitHub
+, stdenv
+, symlinkJoin
+
+, config
+, cudaSupport ? config.cudaSupport
+, cudaPackages ? { }
+
+, rocmSupport ? false
+, hip
+, hipblas
+, rocblas
```

Comment:
hip = throw "'hip' has been removed in favor of 'rocmPackages.clr'"; # Added 2023-10-08

Reply:
Thanks, fixed.